core/camel-management/src/test/java/org/apache/camel/management/ManagedPooledExchangeTest.java

```diff
@@ -59,9 +60,7 @@ protected CamelContext createCamelContext() throws Exception {
     @Test
     public void testSameExchange() throws Exception {
         // JMX tests dont work well on AIX CI servers (hangs them)
-        if (isPlatform("aix")) {
-            return;
-        }
+        assumeFalse(isPlatform("aix"));
```

Comment:
I'm wondering if it is needed as the class is annotated with `@DisabledOnOs(OS.AIX)`

Reply:
Excellent note! This annotation is much better, than assumption. I replace it there possible